src/renderer/components/status-bar/status-bar-items.injectable.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import type { IComputedValue } from "mobx";
+import { computed } from "mobx";
+import type { StatusBarItemProps } from "./status-bar-registration";
+import type { StatusBarItem } from "./status-bar-item-injection-token";
+import { statusBarItemInjectionToken } from "./status-bar-item-injection-token";
+import { computedInjectManyInjectable } from "@ogre-tools/injectable-extension-for-mobx";
+
+export interface StatusBarItems {
+  right: React.ComponentType<StatusBarItemProps>[];
+  left: React.ComponentType<StatusBarItemProps>[];
+}
+
+interface Dependencies {
+  registrations: IComputedValue<StatusBarItem[]>;
+}
+
+function getStatusBarItems({ registrations }: Dependencies): IComputedValue<StatusBarItems> {
+  return computed(() => {
+    const res: StatusBarItems = {
+      left: [],
+      right: [],
+    };
+
+    for (const registration of registrations.get()) {
+      if (!registration || typeof registration !== "object") {
+        continue;
+      }
+
+      const { position = "right", component, visible } = registration;
+
+      if (position !== "left" && position !== "right") {
+        throw new TypeError("StatusBarRegistration.components.position must be either 'right' or 'left'");
+      }
```

Comment:
This part seems to be there to serve Extension API? Our code is protected by typescript, so we don't need it. 

And since it's already present in `registrator`, you can just delete it here.